packages/webchannel-wrapper/tools/build.js

```diff
@@ -28,7 +33,44 @@ closureBuilder.build({
       output_wrapper:
         "(function() {%output%}).call(typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : typeof window !== 'undefined' ? window : {})",
       language_out: 'ECMASCRIPT5',
-      compilation_level: 'ADVANCED_OPTIMIZATIONS'
+      compilation_level: 'ADVANCED'
```

Comment:
Interesting.  Were we not getting advanced optimizations before then?  Or does either value work?

Reply:
yep, either one works, but only ADVANCED is listed in their latest documentation. https://github.com/google/closure-compiler/wiki/Flags-and-Options